pkg/v1/nodegroup/requests_opts.go

```diff
@@ -53,6 +53,9 @@ type CreateOpts struct {
 
 	// AutoscaleMaxNodes represents maximum possible number of worker nodes in the nodegroup.
 	AutoscaleMaxNodes *int `json:"autoscale_max_nodes,omitempty"`
+
+	// UserData represents base64 data which is used to pass a shell script that worker nodes run on boot.
```

Comment:
Let's make this description more common, cause user-data can be as s python or perl script.